docs/fsharp/style-guide/component-design-guidelines.md

```diff
@@ -5,7 +5,7 @@ ms.date: 05/14/2018
 ---
 # F# component design guidelines
 
-This document is a set of component design guidelines for F# programming, based on the F# Component Design Guidelines, v14, Microsoft Research, and [another version](https://fsharp.org/specs/component-design-guidelines/) originally curated and maintained by the F# Software Foundation.
```

Comment:
@cartermp I removed this broken link completely because the component design guidelines link on https://fsharp.org/specs/ points back to this article. Can you approve?

Reply:
Sounds good